README.md

```diff
@@ -80,12 +80,13 @@ void initFBDeferredDeeplinks() async {
   try {
 
     deepLinkUrl = await FlutterFacebookAppLinks.initFBLinks();
-    if(Platform.isIOS)
-      deepLinkUrl = await FlutterFacebookAppLinks.getDeepLink();
+    if(Platform.isIOS) {
+    deepLinkUrl = await FlutterFacebookAppLinks.getDeepLink();
+    }
```

Comment:
Here, I've fixed the warning that statements should be wrapped inside of blocks.